torch/masked/maskedtensor/core.py

```diff
@@ -342,7 +342,7 @@ def from_values(data, mask):
         class Constructor(torch.autograd.Function):
             @staticmethod
             def forward(ctx, data, mask):
-                return MaskedTensor(data.clone(), mask.clone())
+                return MaskedTensor(data, mask)
```

Comment:
Do we want to keep around `from_values`?

Reply:
I'm okay either way -- I suppose we could remove it, but I suppose also no harm. I suppose the more disciplined way would just have `masked_tensor` call `MaskedTensor()` and then `as_masked_tensor` call `MaskedTensor.from_values` -- I'll change it to this if we're all okay with that?